tests/phpunit/integration/Core/Dismissals/REST_Dismissals_ControllerTest.php

```diff
@@ -83,6 +83,107 @@ public function test_get_dismissed_items() {
 		);
 	}
 
+
+	/**
+	 * @dataProvider provider_dismissed_item_deletions
+	 */
+	public function test_delete_dismissed_items( $slugs_to_delete, $expected_remaining_items ) {
+		remove_all_filters( 'googlesitekit_rest_routes' );
+		$this->controller->register();
+		$this->register_rest_routes();
+
+		$this->dismissed_items->add( 'foo' );
+		$this->dismissed_items->add( 'bar', 100 );
+		$this->dismissed_items->add( 'baz', 200 );
+
+		$this->assertEqualSets(
+			array( 'foo', 'bar', 'baz' ),
+			array_keys( $this->dismissed_items->get() )
+		);
+
+		$request = new WP_REST_Request( 'DELETE', '/' . REST_Routes::REST_ROOT . '/core/user/data/dismissed-items' );
+		$request->set_body_params(
+			array(
+				'data' => array(
+					'slugs' => $slugs_to_delete,
+				),
+			)
+		);
+		$response = rest_get_server()->dispatch( $request );
+
+		$this->assertEqualSets(
+			$expected_remaining_items,
+			array_keys( $this->dismissed_items->get() )
+		);
+
+		$this->assertEqualSets(
+			$expected_remaining_items,
+			$response->get_data()
+		);
+	}
+
+
+	public function provider_dismissed_item_deletions() {
+		return array(
+			'slugs as single item string'     => array(
+				'foo', // Slugs to delete.
```

Comment:
It's worth noting that if a string is passed for `slugs` it will be coerced to an array, this is the expected behaviour as discussed under [Primitive Types > Type Juggling](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/#primitive-types) in the WordPress docs.